### DISCUSSION TITLE
Unable to provide data/files related to discussions using the upload features on GitHub.

### DISCUSSION THREAD

**Discussion by jwraby** [dtcenter/METplus#2466] [Incoming] [answered]
The upload functionality available on GitHub when submitting a discussion item does not work for me. I suspect that this is due to network restrictions imposed by the Army. I tried using the instructions for FTP at https://github.com/dtcenter/METplus/discussions/954, but this doesn't work either due to the restrictions. Do you have any suggestions for an alternate method for delivering files/data to you. I intend to follow-up on a prior discussion with Tara Jensen about some new observations data I would like to work with in METplus, but this will be difficult without the ability to deliver examples of the observations. Thanks.

**Comment by jprestop** [comment]
Hi @jwraby.  I'm sorry to learn of the troubles you've been having trying to get data to us.  In the past, you have used email or uploaded it to a government file server and provide a link for us to download the files.  Is that a feasible way to proceed with the data that you'd like to get to us?

**Comment by jwraby** [comment]
Hi Julie - 

 

Thanks for your response. Yes. Having an email address has worked in the
past for me. I post the files to the server and then the server uses the
email address to send you a link to download the files.

 

R/

John

 

 

 

From: Julie Prestopnik ***@***.***> 
Sent: Tuesday, January 16, 2024 11:54 AM
To: dtcenter/METplus ***@***.***>
Cc: Raby, John W CIV USARMY DEVCOM ARL (USA) ***@***.***>;
Mention ***@***.***>
Subject: [Non-DoD Source] Re: [dtcenter/METplus] Unable to provide
data/files related to discussions using the upload features on GitHub.
(Discussion #2466)

 

Hi @jwraby
<https://usg01.safelinks.protection.office365.us/?url=https%3A%2F%2Fgithub.c
om%2Fjwraby&data=05%7C02%7Cjohn.w.raby2.civ%40army.mil%7C193f3281f4704048774
908dc16c48eeb%7Cfae6d70f954b481192b60530d6f84c43%7C0%7C0%7C63841028068696373
1%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haW
wiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=RrKY%2FK9rxiMLPlT%2FmDN%2F01sMiEK2yCj
2teOqsJzUHy0%3D&reserved=0> . I'm sorry to learn of the troubles you've been
having trying to get data to us. In the past, you have used email or
uploaded it to a government file server and provide a link for us to
download the files. Is that a feasible way to proceed with the data that
you'd like to get to us?

-
Reply to this email directly, view it on GitHub
<https://usg01.safelinks.protection.office365.us/?url=https%3A%2F%2Fgithub.c
om%2Fdtcenter%2FMETplus%2Fdiscussions%2F2466%23discussioncomment-8148184&dat
a=05%7C02%7Cjohn.w.raby2.civ%40army.mil%7C193f3281f4704048774908dc16c48eeb%7
Cfae6d70f954b481192b60530d6f84c43%7C0%7C0%7C638410280686963731%7CUnknown%7CT
WFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3
D%7C3000%7C%7C%7C&sdata=vynAV%2Fu0%2FHGH5Koa8LdFHAZmaNKB8NYCmJCBMpUiDSE%3D&r
eserved=0> , or unsubscribe.
You are receiving this because you were mentioned.
<https://github.com/notifications/beacon/AUX5FH7C5MPHSVLWGQABZUTYO3EC7A5CNFS
M6AAAAABB5DZ7RGWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3T
UL5UWJTQAPRKNQ.gif> Message ID:
***@***.***>

**Reply by jprestop** [reply, marked as answer]
Great!  Thanks, @jwraby!  I just sent you an email with my email address, which you can use, and I'll grab the files.

Regarding:

> I intend to follow-up on a prior discussion with Tara Jensen about some new observations data I would like to work with in METplus

Do you know if there is a METplus GitHub issue created for this work that I can refer to, and if so, can you please pass it along so that I can keep all of the information and data together in an organized place?  If not, do you have another contact (as Tara is out of the virtual office) whom I should pass this data along to?

**Reply by jprestop** [reply]
Thanks @jwraby for marking this as answered.  I just wanted to follow up and confirm that I did receive the test file from you and mention that we have been corresponding via email about where to put this data.  I will note that you plan to submit a Discussion about this data, which will potentially be used to create a new METplus Use Case and that this potential use case came out of discussion at the METplus Advanced Training Series from 15 NOV 2023.  I'll go ahead and lock this Discussion to prevent future posts here.  We look forward to your new Discussion about the potential new use case.
